src/task/task.ts

```diff
@@ -101,12 +103,27 @@ export default class TaskImpl<T> implements Task<T> {
     );
   }
 
-  async enable(): Promise<void> {
+  async enable(options?: TaskEnablingOptions): Promise<void> {
+    let patchOp = async () => this._data.patch(() => ({ enabled: true }));
+
+    // Patch task with enabling options
+    if (options) {
+      if (!!options.skipPastOccurrences && this._data.nextRunTime) {
```

Comment:
can be collapsed into a single condition

```suggestion
    if (options && options.skipPastOccurrences && this._data.nextRunTime) {
    }
```